### PR TITLE
fix(screenshot-capture) Update screenshot capture feature

### PR DESCRIPTION
**Pull request by robertpin**
Add participants jid list to request

Enable screenshot capture only when recording is also on

### CODE REVIEW

**Comment by robertpin** [comment]
Jenkins test this please

File: react/features/screenshot-capture/functions.js [hunk 1]
```diff
@@ -18,3 +19,26 @@ export function createScreenshotCaptureSummary(stateful: Object | Function) {
...
+
+/**
+ * Get a participant's connection JID given its ID.
+ *
+ * @param {Object} state - The redux store state.
+ * @param {string} participantId - ID of the given participant.
+ * @returns {string|undefined} - The participant connection JID if found.
+ */
+export function getParticipantJid(state: Object, participantId: string) {
+    const conference = getCurrentConference(state);
```
Comment (quitrk) [inline react/features/screenshot-capture/functions.js:31]: nit
```
getCurrentConference(state)?
 .getParticipantById(participantId)?
 .getJid()
```

**Review by quitrk — COMMENTED** [review]
It seems that there's a lot of calls to `toggleScreenshotCaptureSummary`. Can't we instead react to screensharing state change through a middleware?

File: react/features/screenshot-capture/functions.js (hunk 1 above)
Comment (robertpin) [reply react/features/screenshot-capture/functions.js:31]: "Flow does not yet support method or property calls in optional chains."

**Comment by robertpin** [comment]
> It seems that there are a lot of calls to `toggleScreenshotCaptureSummary`. Can't we instead react to screensharing state change through a middleware?

The thing is we want to enable it only when screensharing and recording are on at the same time

**Review by quitrk — DISMISSED** [review]

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Review by quitrk — APPROVED** [review]

**Pull request merged**